### ISSUE TITLE
call voice does not work

### ISSUE THREAD

**Issue by khanhbgpro** [BTNGaming/btn_gcphone#21]
I installed and completed the call
 but I couldn't hear the voice from the person I called, please help me

**Comment by Heady20006** [comment]
can you hear people without the phone?

**Comment by Caiolib** [comment]
me call voice is not working too, and yes can hear without phone

**Comment by henrykobutra** [comment]
Same here, everything works except voice when calling.

**Comment by Ashember** [comment]
Yes Same, Every thing seems to be working fine except the Phone Calls the other person, and they do not receive the ring or the text.   it appears to work when you call from your phone.. it rings and such, Text messages seem to appear to send, though the other person never receives either communication.   

If anyone finds a fix for this please post it.

**Comment by mick340** [comment]
If you use VMenu that will override it. You need to disable it

**Comment by murphyhf** [comment]
same here
